Liubomyr_Kopets/ToDo/src/app/todocomponent/todocomponent.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-todocomponent',
+  templateUrl: './todocomponent.component.html',
+  styleUrls: ['./todocomponent.component.scss']
+})
+export class TodocomponentComponent {
+  @Input() items: string;
+  @Output() editItem = new EventEmitter<any>();
```

Comment:
Avoid explicit setting the "any" type. If the event doesn't emit any value you can set "void" type